internal/host/base_host_run.go

```diff
@@ -668,7 +668,11 @@ func (h BaseHostRun) ReadFile(ctx context.Context, name string) (io.ReadCloser,
 		}
 	}
 
-	stdoutReader, stdoutWriter := io.Pipe()
+	stdoutReader, stdoutWriter, err := os.Pipe()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
I got some weird race conditions detected within the pipes from `io.Pipe()` at `HostStore` which can't be reproduced when using `os.Pipe()`, so... changing this to `os.Pipe()` just in case.